client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -255,6 +272,7 @@ const mapDispatchToProps = ( dispatch, { activityId, siteId } ) => ( {
 			)
 		)
 	),
+	getHelpClick: () => recordTracksEvent( 'calypso_activitylog_threat_get_help' ),
```

Comment:
if we are going to record this we probably want to record what kind of threat it was. we can do that by passing a payload to the tracks event as a second parameter

Reply:
Sure, makes sense, but I think it might be more helpful to pass in the `activityId`. This way it's independently identifiable if there are multiple of the same threat in the log. Thoughts?

Reply:
can send both. my initial thought was that if we wanted to monitor specific threats we could bubble them up by type and see "what are the top 10 threats our customers are seeing"

Reply:
Can you suggest how I might get access to what type of threat it is? The `activity`
 object doesn't seem to contain anything except the string description.

Reply:
not sure. we could add the security threat info as some `additionalData` property when transforming from the API. that could make sense, then we just add `activityItem.additionalData.threatSignature` to the tracks event.

Do what you feel right about.